components/ILIAS/IndividualAssessment/PRIVACY.md

```diff
@@ -0,0 +1,116 @@
+# Individual Assessment Privacy
+
+Disclaimer: This documentation does not warrant completeness or correctness. Please
+report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+### General Information
+
+The Individual Assessment is intended as a performance and skill evaluation tool.
+The submitted data can shape a person's career. Therefore, some data is intentionally
+impossible to delete or change for most or all user roles. This makes it more likely
+that Individual Assessment records are acceptable as proof in court.
+
+### Integrated Services
+
+- The Individual Assessment component employs the following services, please consult
+the respective privacy.mds:
+    - [Metadata](../MetaData/Privacy.md)
+    - [AccessControl](../AccessControl/PRIVACY.md)
+    - [Object](../ILIASObject/PRIVACY.md)
+    - [InfoScreen](../InfoScreen/PRIVACY.md)
+- Users from [Course](../Course/PRIVACY.md) and [Group](../Group/PRIVACY.md) can be
+  added to an Individual Assessment. Even after a person's membership to the original
+  object has been removed, it may be obvious where the Users came from.
+- An Individual Assessment may be part of another object's Learning Progress
+  [Tracking](../Tracking/PRIVACY.md).
+
+## Data being stored
+
+- **Participants**: Adding users as Participants to the Individual Assessment references
+  their User object by ID.
+- **Examiner**: Examiners are users with the permission to grade a Participant. Their
+  User ID is stored with the individual record.
+- **Location, time and date of an assessment**: The Examiner can enter when and where
+  an exam took place. These fields can be set as required in the Settings.
+- **Grading**: The Examiner selects whether the Participant completed or failed the
+  assessment. Grading might influence the overall Learning Progress of another object.
+- **Record Notes**: Examiners can write notes - both public to the individual participant
```

Comment:
Byline says: "If notification is enabled below, the record note will be send to the participant by e-mail, too." Does it mean paticipant's e-mail-addresses are stored too? It wasn't mentioned under **Participants**

Reply:
Good question! 
And what about Internal record notes? They are indeed only visible to Tutors, but aren't they stored as well?

Reply:
So, no, mail addresses are not stored. They are taken from the users account that is referenced by ID. And yes, the internal note is also stored. And I think we say so in the according section: 

> **Record Notes**: Examiners can write notes - both public to the individual participant and internal notes only visible to users with specific permissions.

If any of this should be clarified I would be happy about a proposal. From my POV: I would not clarify that the mail address is not stored. I think we should only mention things that are in fact stored. The list of things that are not stored is huge (e.g. firstname, lastname, ...). If we mention one thing that is not stored explicitly this IMHO leaves more ambiguity for the other things that are not stored. Regarding the internal note I think we could clarify if the current text is not clear enough, but for me it is ok.

Reply:
Thanks for the explanation @klees. Can you then explicitly say "internal record notes" instead of "Internat notes"?

Reply:
Done!